openmc/mgxs/__init__.py

```diff
@@ -6,12 +6,18 @@
 from openmc.mgxs.mdgxs import *
 
 GROUP_STRUCTURES = {}
-"""Dictionary of commonly used energy group structures, including "CASMO-X" (where X
-is 2, 4, 8, 16, 25, 40 or 70) from the CASMO_ lattice physics code and other commonly
-used activation_ energy group structures "VITAMIN-J-175", "TRIPOLI-315", "CCFE-709_" 
-and "UKAEA-1102_"
+"""Dictionary of commonly used energy group structures:
+- "CASMO-X" (where X is 2, 4, 8, 16, 25, 40 or 70) from the CASMO_ lattice
+physics code
+- "XMAS-172" designed for LWR analysis (Sartori 1990, Santamarina et al. 2004)
+- "SHEM-361" designed for LWR analysis to eliminate self-shielding calculations
```

Comment:
Need to make this `SHEM-361_` to pick up the link below

Reply:
ok thanks I was not sure what the '''_''' was for